processing/src/main/java/io/druid/query/groupby/epinephelinae/ConcurrentGrouper.java

```diff
@@ -80,7 +90,9 @@ public ConcurrentGrouper(
       final ObjectMapper spillMapper,
       final int concurrencyHint,
       final DefaultLimitSpec limitSpec,
-      final boolean sortHasNonGroupingFields
+      final boolean sortHasNonGroupingFields,
+      @Nullable final ListeningExecutorService grouperSorter,
```

Comment:
It is null when brokers merge subquery results for nested groupBys. I'll add a test.